notebooks/book2/03/lkj_numpyro.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        R = dist.LKJ(dimension=2, concentration=eta).sample(jax.random.PRNGKey(0), (int(1e4),))
</pre><p>It may be good to add a comment on what the second parameter is and why you have kept it to 1e4</p><p>Can we choose a more informative variable name rather than R</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>